src/middleware/secure-headers/index.test.ts

```diff
@@ -0,0 +1,154 @@
+import { Hono } from '../../hono'
```

Comment:
You can add the follow line on the top to fix lint errors:

```ts
/* eslint-disable quotes */
```

This allow using double quotes.

Reply:
thank you !! fix it.